src/components/Bill.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  BillContainer,
+  BillSelectionContainer,
+  SelectTipContainer,
+  SelectTipGrid,
+  SelectTipItems,
+  Title,
+} from './Bill.styled';
+import InputField from './InputField/InputField';
+import dollarSign from '../images/icon-dollar.svg';
+import personIcon from '../images/icon-person.svg';
+import { TipInput } from './Bill.styled';
+import { Text } from './Text/Text';
+import colors from './colors';
+export const Bill = () => {
+  return (
+    <BillContainer>
+      <BillSelectionContainer>
+        <InputField
+          title="Bill"
+          defaultValue="0"
+          type="number"
+          icon={dollarSign}
+        />
+        <SelectTipContainer>
+          <Text padding="0 0 10px 0" color={colors.neutral.darkGrayishCyan}>
+            Select Tip %
+          </Text>
+          <SelectTipGrid>
+            <SelectTipItems>
+              <Text size="1.5rem">5%</Text>
```

Comment:
The `size="1.5rem"` can be refactored as a styled component and resued, instead of having it all over the place